### PR TITLE
fix: batching logic to handle single block edge case

### PR DESCRIPTION
**Pull request by prkpndy**
## Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

## What is the current behavior?

There is an edge case for SNOS batching with a single block.

Resolves: #NA

## What is the new behavior?

All batches are created properly. Update the response structure for bouncer weights.

### CODE REVIEW

**Review by Mohiiit — APPROVED** [review]
LGTM!

File: orchestrator/src/worker/event_handler/triggers/batching.rs [hunk 1]
```diff
@@ -547,7 +547,7 @@ impl BatchingTrigger {
         if self.should_close_snos_batch(config, &current_snos_batch).await? {
             // Close the current batch and start a new batch
             self.update_or_create_snos_batch_in_db(&current_snos_batch, config, SnosBatchStatus::Closed).await?;
-            self.start_snos_batch(current_snos_batch.snos_batch_id + 1, None, block_number)
+            self.start_snos_batch(current_snos_batch.snos_batch_id + 1, None, current_snos_batch.end_block + 1)
```
Comment (apoorvsadana) [inline orchestrator/src/worker/event_handler/triggers/batching.rs:550]: no change for l2s?

Comment (prkpndy) [reply orchestrator/src/worker/event_handler/triggers/batching.rs:550]: yeah, L2s don't require changes since there the logic is different (should close batch is checked for one block before - so it should be fine. have confirmed this as well with edge cases. will test e2e again after L2s is fixed to handle Fusaka upgrade)

**Review by apoorvsadana — APPROVED** [review]
